### PR TITLE
Fix login portal

### PR DESCRIPTION
**Pull request by ghost**
## Description
Fix login portal
## Reference Requirements
- Product Backlog ID: Non-PBI
- Task ID: Non

## Reference Isuees
#{issue_id}

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -68,9 +68,15 @@ public function login(LoginFormValidation $request)
...
-
-            if ($portalResponse->meta->status == config('define.login.msg_success')) {
-                $user = $this->saveUser($portalResponse, $request);
+            if ($portalResponse->access_token) {
+                $access_token = $portalResponse->access_token;
+                $portalUserProfiles = $client->request('GET', config('portal.base_url_api') . config('portal.end_point.user_profiles'),[
+                    'headers' => [
+                        'authorization' => $access_token,
+                        ] 
+                    ]);
```
Comment (at-caonguyen) [inline app/Http/Controllers/Auth/LoginController.php]: Sai convention nè

File: app/Http/Controllers/Auth/LoginController.php [hunk 2]
```diff
@@ -68,9 +68,15 @@ public function login(LoginFormValidation $request)
             $client = new Client();
             $portal = $client->post(config('portal.base_url_api') . config('portal.end_point.login'), ['form_params' => $data]);
             $portalResponse = json_decode($portal->getBody()->getContents());
-
-            if ($portalResponse->meta->status == config('define.login.msg_success')) {
-                $user = $this->saveUser($portalResponse, $request);
+            if ($portalResponse->access_token) {
+                $access_token = $portalResponse->access_token;
```
Comment (at-caonguyen) [inline app/Http/Controllers/Auth/LoginController.php]: tên biến sai convention nhé

**Review by mvn-caonguyen-dn — COMMENTED** [review]
check cmt

**Comment by at-caonguyen** [comment]
LGTM :+1:

File: app/Http/Controllers/Auth/LoginController.php [hunk 3]
```diff
@@ -68,9 +68,15 @@ public function login(LoginFormValidation $request)
             $client = new Client();
             $portal = $client->post(config('portal.base_url_api') . config('portal.end_point.login'), ['form_params' => $data]);
             $portalResponse = json_decode($portal->getBody()->getContents());
-
-            if ($portalResponse->meta->status == config('define.login.msg_success')) {
-                $user = $this->saveUser($portalResponse, $request);
+            if ($portalResponse->access_token) {
+                $accessToken = $portalResponse->access_token;
+                $portalUserProfiles = $client->request('GET', config('portal.base_url_api') . config('portal.end_point.user_profiles'), [
```
Comment (at-huydq) [inline app/Http/Controllers/Auth/LoginController.php]: tách cái portal ra thành 1 cái trait Portal giúp a. 
viết 2 phương thức  login  vs get profile trong đó.

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**